db/data/20220322135814_create_bsnl_credential.rb

```diff
@@ -0,0 +1,17 @@
+class CreateBsnlCredential < ActiveRecord::Migration[5.2]
+  require "rake"
+
+  def up
+    rake = Rake.application
+    rake.init
+    rake.add_import "lib/tasks/refresh_bsnl_sms_jwt.rake"
+    rake.load_rakefile
+
+    Credential.create(name: "BSNL_SMS_JWT", value: "")
```

Comment:
Since these aren't env vars, I don't think we need to be so loud. `bsnl_sms_jwt` reads just fine to me :)